.circleci/config.yml.in

```diff
@@ -659,6 +659,7 @@ jobs:
       <<: *environment
       CUDA_VERSION: 11.6
       image_name: pytorch/torchaudio_unittest_base:manylinux-cuda10.2-cudnn8-20210623
+      TORCHAUDIO_TEST_ALLOW_SKIP_IF_CUDA_SMALL_MEMORY: true
```

Comment:
Can you move this under test section? similar to L646-648 above.

Reply:
Next to L680/681

Reply:
fixed in new commit